benefits/enrollment/api.py

```diff
@@ -93,6 +93,7 @@ def __init__(self, response, requested_id, payload=None):
         self.customer_ids = list(payload)
         self.updated_customer_id = self.customer_ids[0] if len(self.customer_ids) == 1 else None
         self.success = requested_id == self.updated_customer_id
+        self.message = "Updated customer_id does not match enrolled customer_id" if not self.success else ""
```

Comment:
Yes, it used to be in the [`_enroll()` view helper](https://github.com/cal-itp/benefits/blob/dev/benefits/enrollment/views.py#L90).

I moved it to the `api` module since the error really is coming from the API call (and the [very specific logic of parsing it](https://github.com/cal-itp/benefits/blob/dev/benefits/enrollment/api.py#L88)), and it felt a little weird to have the _error message_ defined in another place.